### ISSUE TITLE
Histogram returning NaN

### ISSUE THREAD

**Issue by mslabik** [fastify/under-pressure#125]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported

### Fastify version

3.20.1

### Plugin version

5.7.0

### Node.js version

14.17.1

### Operating system

macOS

### Operating system version (i.e. 20.04, 11.3, 10)

10.14.6 (Mojave)

### Description

In some cases, on very high load (or some time-consuming sync actions) the node's `histogram.mean` can be `NaN` and produce a false-negative condition that does not trigger under-pressure. In extreme cases, it can lead to node process being unresponsive. 

```js
Histogram {
  min: 9223372036854776000,
  max: 0,
  mean: NaN,
  exceeds: 0,
  stddev: NaN,
  percentiles: SafeMap(1) [Map] { 100 => 0 }
}
```

The `min` value however is always present and could be used in the comparison instead. The second option is to use `Infinity` to compare

### Steps to Reproduce

Both scenarios were reproduced using `autocannon`: 

a) Time consuming synchronous action used in fastify route handler, e.g.:
```js
function sleep(ms) {
    const start = new Date().getTime();
    const expire = start + ms;
    while (new Date().getTime() < expire) { }
}
```

b) Big amount of connections to real-life service (500 or more in our case)

### Expected Behavior

under-pressure being calculated properly and triggered

**Comment by climba03003** [comment]
`Histogram` is created by `node` core. Can you file a bug report in there?

**Comment by mcollina** [comment]
I think we should check if the histogram is returning `NaN` and trigger the condition anwyay.

**Comment by climba03003** [comment]
The fix would be similar to
```js
eventLoopDelay = Math.max(0, histogram.mean / 1e6 - resolution)
// we assume the delay is too large that break `perf_hooks`, so it would be Infinity
if(isNaN(eventLoopDelay)) eventLoopDelay  = Infinity 
```

**Comment by mslabik** [comment]
Should I create a PR with that fix?

**Comment by climba03003** [comment]
> Should I create a PR with that fix? 

Yes, please

**Comment by mslabik** [comment]
PR: https://github.com/fastify/under-pressure/pull/126

**Comment by mcollina** [comment]
Fixed in v5.8.0
